### ISSUE TITLE
Project Deploy Preview does not support --concise flag

### ISSUE THREAD

**Issue by petter-eikeland** [forcedotcom/cli#2645]
### Summary

`sf project deploy preview` does not support --concise flag [as per release notes](https://github.com/forcedotcom/cli/blob/main/releasenotes/README.md#2227-jan-3-2023). However, `sf project retrieve preview` does support the flag. The --concise flag would be really useful for larger projects with many files ignored :)

### Steps To Reproduce
1.  Create a scratch org
1.  Execute `sf project deploy preview --concise` 

### Expected result

Output showing all files to be deployed, omitting forceignored files

### Actual result

```
sf project deploy preview --concise                       
Error (2): Nonexistent flag: --concise
```

### System Information

```json
{
  "architecture": "darwin-arm64",
  "cliVersion": "@salesforce/cli/2.24.4",
  "nodeVersion": "node-v20.10.0",
  "osVersion": "Darwin 22.6.0",
  "rootPath": "/Users/user/.local/share/sf/client/2.24.4-c9898a2",
  "shell": "zsh",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 3.0.5 (core)",
    "@oclif/plugin-commands 3.1.0 (core)",
    "@oclif/plugin-help 6.0.10 (core)",
    "@oclif/plugin-not-found 3.0.8 (core)",
    "@oclif/plugin-plugins 4.1.15 (core)",
    "@oclif/plugin-search 1.0.11 (core)",
    "@oclif/plugin-update 4.1.7 (core)",
    "@oclif/plugin-version 2.0.11 (core)",
    "@oclif/plugin-warn-if-update-available 3.0.9 (core)",
    "@oclif/plugin-which 3.0.14 (core)",
    "@salesforce/cli 2.24.4 (core)",
    "1commerce 0.0.10 (user)",
    "apex 3.0.17 (core)",
    "auth 3.2.0 (core)",
    "community 3.0.12 (user)",
    "data 3.0.14 (core)",
    "deploy-retrieve 3.1.0 (core)",
    "info 3.0.15 (core)",
    "limits 3.1.2 (core)",
    "marketplace 1.0.16 (core)",
    "org 3.2.2 (core)",
    "packaging 2.1.0 (user)",
    "schema 3.1.0 (core)",
    "settings 2.0.15 (core)",
    "signups 2.0.15 (user)",
    "sobject 1.1.0 (core)",
    "source 3.1.3 (core)",
    "telemetry 3.1.9 (core)",
    "templates 56.0.9 (core)",
    "trust 3.3.1 (core)",
    "user 3.2.1 (core)",
    "@salesforce/sfdx-plugin-lwc-test 1.1.1 (user)",
    "sfdmu 4.32.2 (user)",
    "sfdx-browserforce-plugin 4.0.0 (user)"
  ]
}
```

**Comment by shetzel** [comment]
There's already a PR open for it.  After successful review and QA, should be in the next CLI release candidate.  https://github.com/salesforcecli/plugin-deploy-retrieve/pull/873

**Comment by shetzel** [comment]
The fix is now in the sf CLI release candidate v2.25.7.

**Comment by petter-eikeland** [comment]
@shetzel 
It looks good! Thank you for this enhancement 😊
